src/mono/wasm/debugger/debugger.md

```diff
@@ -0,0 +1,23 @@
+# Mono Wasm Debugger
+
+## Native debugging
+
+It's possible to debug native code and managed code using BrowserDebugProxy.
+
+Steps:
+- Install C/C++ DevTools Support (DWARF) extension on chrome.
+- Enable DWARF support: Open DevTools, click on the Engine, click on experiments and enable WebAssembly Debugging: Enable DWARF support.
+- Start the WebAssembly App Without Debugging from VS or dotnet run on command line.
+- Run chrome using this startup parameter: --remote-debugging-port=9222
+- Go to your Blazor App Page
+- Press Ctrl-Alt-D (windows) it will open the debugger page
+- It will show something like this in the Address Bar: ``http://localhost:9222/devtools/inspector.html?ws=127.0.0.1:9300/devtools/page/97FCDA5A332CA3B72031790B26A264EF``
```

Comment:
Is this manual process needed for native debugging specifically?

And does this work with `dotnet run --debug` for a template browser app too?